projects/i18n/languages/russian/core.ts

```diff
@@ -1,5 +1,5 @@
 import {LanguageCore} from '@taiga-ui/i18n/interfaces';
-import {TUI_RUSSIAN_LANGUAGE_COUNTRIES} from './countries';
+import {TUI_RUSSIAN_LANGUAGE_COUNTRIES} from '@taiga-ui/i18n/languages/russian';
```

Comment:
This should be changed back to the way it was

Reply:
Done